apps/sms/index.html

```diff
@@ -128,7 +129,7 @@
 
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
-      <section id="thread-list" class="panel default-panel" role="region">
+      <section class="panel panel-thread-list" role="region">
```

Comment:
maybe we can start using ```panel-inbox```/```panel-conversation```? Or even switch from ```panel``` to ```view``` like we do for folder structure and JS objects?